db/migrations/20210106133650_create-residents.js

```diff
@@ -0,0 +1,27 @@
+exports.up = function(knex) {
+  return knex.schema
+  .createTable('residents', function (table) {
+    table.increments('id').primary();
+    table.string('name');
```

Comment:
`table.string` make sense. More research is needed to understand seeding more complex data types.